fines/models.py

```diff
@@ -1,11 +0,0 @@
-from django.db import models
-
-from loans.models import Loan
-
-
-# Create your models here.
-class Fine(models.Model):
```

Comment:
need to add class Meta
add relevant helping + error messages
add __str__ function for the admin